<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../design/wheel.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="prize_l">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PRIZE: </string>
   </property>
  </widget>
  <widget class="QLabel" name="spin_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../design/spin_btn.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="guess_txt">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="word_l">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>WORD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cat_l">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>CATEGORY:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Letter/Phrase</string>
   </property>
  </widget>
  <widget class="QLabel" name="turn_l">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TURN:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="phrase_checkbox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Guess whole phrase</string>
   </property>
  </widget>
  <widget class="QLabel" name="guess_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../design/guess_btn_disabled.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="guessed_l">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>GUESSED: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="balance_l">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 0)</string>
   </property>
   <property name="text">
    <string>BALANCE (PLAYER):</string>
   </property>
  </widget>
  <widget class="QLabel" name="letter_error_msg">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Error message</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_l_cpu">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>BALANCE (CPU):</string>
   </property>
  </widget>
  <widget class="QLabel" name="hits_l">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 0)</string>
   </property>
   <property name="text">
    <string>HITS (PLAYER):</string>
   </property>
  </widget>
  <widget class="QLabel" name="hits_l_cpu">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>HITS (CPU):</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
